frontend/public/components/RBAC/edit-rule.jsx

```diff
@@ -503,9 +512,11 @@ const EditRule = connect(stateToProps, { getResources: k8sActions.getResources }
               <div className="col-xs-2" />
               <div className="col-xs-10">
                 <HRMinor />
-                <label htmlFor="api-groups">API Groups</label>
+                <label htmlFor="api-groups">{t('edit-rule~API Groups')}</label>
                 <p className="text-secondary">
-                  Restrict this role to a subset of API URLs that don&rsquo;t correspond to objects.
+                  {t(
+                    'edit-rule~Restrict this role to a subset of API URLs that don&rsquo;t correspond to objects.',
```

Comment:
Remove HTML entities (`&rsquo;`) from message content.

Reply:
Changed \&rsquo; to ' in file.Thanks.

Reply:
<p className="text-secondary">
                  {t(
                    "edit-rule~Restrict this role to a subset of API URLs that don't correspond to objects.",
                  )}
                </p>